starters/lit-mobx-vaadin/web-dev-server.config.mjs

```diff
@@ -0,0 +1,26 @@
+// import { hmrPlugin, presets } from '@open-wc/dev-server-hmr';
+
+/** Use Hot Module replacement by adding --hmr to the start command */
+const hmr = process.argv.includes('--hmr');
```

Comment:
I believe @open-wc/dev-server-hmr is still experimental, i think we are discouraged of using anything that's still experimental/beta/unreleased. we should probably remove it.